src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -53,10 +53,10 @@ public UserDto createUser(CreateUserRequest request) {
     }
 
     public UserDto deleteUser(DeleteUserRequest request) {
```

Comment:
I'm not sure there's a good way to check for whether a user is in a room inside the service class. It doesn't make sense to pass in a roomId since the user is not meant to be connected to the room by the time it's called. `User` objects have a `room` field in the database, but this disappears when the object is mapped to a UserDto for the DeleteUserRequest. I think it may be best just to check for whether a user is connected to a room wherever the function is being called. What do you think?